src/content/docs/bot-protection/intro.mdx

```diff
@@ -0,0 +1,87 @@
+---
+description: "Introduction to Arcjet Bot protection."
+prev: false
+title: "Bot protection Intro"
+---
+
+import { Badge } from "@astrojs/starlight/components";
+import Comments from "/src/components/Comments.astro";
+
+With Arcjet Bot protection you can manage traffic by automated clients and bots.
+
+Bots can be good or bad.
+Good bots identify themselves in a consistent manner so it is straightforward
+to identify them,
+but those are often not the bots you wish to block!
+Bad bots pretend to be real clients and use various mechanisms to evade bot
+detection.
+In an ideal world,
+all bots would respect the `robots.txt` file,
+but unfortunately this is not always the case.
+
+Other platforms also provide bot protection.
```

Comment:
I think we probably want to avoid leading with this 😅

"Arcjet Bot protection is different because..."